third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -2840,6 +2879,34 @@ func cidrOrSizeDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return strings.HasPrefix(new, "/") && strings.HasSuffix(old, new)
 }
 
+// We want to suppress diffs for the scopes automatically added by GKE
+func containerClusterAddedScopesSuppress(k, old, new string, d *schema.ResourceData) bool {
+	o, n := d.GetChange("cluster_autoscaling.0.auto_provisioning_defaults.0.oauth_scopes")
+
+	addedScopes := []string{
```

Comment:
I left this as a slice, for future cases if something were to be added to it, but I can definitely change it to just this value if that's preferred.